### ISSUE TITLE
[Feature] Add A4 paper format

### ISSUE THREAD

**Issue by guppy42** [stolksdorf/naturalcrit#57]
**Browser Type/Version** : Chrome  48.0.2564.116 (64-bit)

**Operating System** : LMDE 2

**Issue Description** : 

Page size is "Letter" - for generating PDF's this is not an issue but when printing it's a bit anoying having to deal with the extra bare strip at the bottom of each page ( we can't buy the "Letter" size paper in europe as far as I know )

A4 is   210mm x 297 mm  - how ever altering the .phb class to this manually adds a blank page after every page ?

**Markdown code to reproduce** :

```
<div class='pageNumber'>1</div>
<div class='footnote'>PART 1 | FANCINESS</div>

\page
```

then attempt to print and select paper size a4 ( click "more settings" in chrome printer dialog to see it )

**Comment by stolksdorf** [comment]
Added as a snippet in v1.3.1

**Comment by guppy42** [comment]
Thanks, how ever it doesn't solve the issue completely - as noted in the original report simply altering the .phb class creates a problem whever every other page is blank when printing (and only when printing) if the content of the page is larger than US Letter  size. if the content will fit on a US Letter page the background image stops where the US Letter sheet would if put on top of a A4 Sheet.

This leads me to belive that there may be some 

`@media print{ }`

trickery going on how ever I see no evidence of this in the generated css

I've prepared an example here;

http://www.naturalcrit.com/homebrew/print/VyigyRlnl

ps. Chrome 49.0.2623.75 (64-bit stable) was added to the LDME repo today  and it supports column printing \o/
